.github/workflows/unittest.yml

```diff
@@ -0,0 +1,109 @@
+name: "UnitTests"
+
+on:
+  # add push to main trigger for sonarcloud scan
+  push:
+    branches:
+      - "main"
+      - "master"
+
```

Comment:
Same question here: ATM the git project uses `master` as the trunk branch. Did you add `main` just anticipating the future change of the trunk branch name from `master` to `main`?